src/server.c

```diff
@@ -3194,14 +3192,13 @@ void initServerConfig(void) {
 
     unsigned int lruclock = getLRUClock();
     atomicSet(server.lruclock,lruclock);
-    resetServerSaveParams();
 
+    resetServerSaveParams();
```

Comment:
Since this in non-functional, please avoid touching stuff to make the git blame easier to follow.